src/app/testing/testing.component.ts

```diff
@@ -130,4 +130,7 @@ export class TestingComponent implements AfterViewInit {
       this.switchBackground();
     }
   }
+  startEnd(): void {
+    this.matchData.roundPhase = "end";
+  }
```

Comment:
Do we not want to move this to changeRoundPhase()?

Reply:
resolved in [839f0b4](https://github.com/ValoSpectra/Spectra-Frontend/pull/11/commits/839f0b48593bd80a3ad25991d52121eb1711c00d)